.github/workflows/breaking-change-alert.yaml

```diff
@@ -53,24 +53,51 @@ jobs:
   notify-breaking-changes:
     runs-on: ubuntu-latest
     steps:
+      - name: Escape inputs for JSON
+        id: escape
+        run: |
+          escape_json() {
+            echo "$1" | jq -R -c .
```

Comment:
This adds quotes around everything that gets sanitized. We need to remove the quotes from the beginning and end.